x/batching/keeper/endblock.go

```diff
@@ -142,7 +165,7 @@ func (k Keeper) ConstructValidatorTree(ctx sdk.Context) ([][]byte, []byte, error
 			}
 			panic(err)
 		}
-		pkBytes, err := decompressPubKey(secp256k1PubKey.Bytes())
+		addr, err := utils.PubKeyToAddress(secp256k1PubKey.Bytes())
```

Comment:
I'm probably being overly verbose but I'd consider naming this something like `ethAddr`, since in this Cosmos context it's reasonable to expect it to be a bech32 address.

Reply:
good idea, I also changed the method name to `PubKeyToEthAddress`